strings/base_error.h

```diff
@@ -529,6 +529,28 @@ WINRT_EXPORT namespace winrt
         return pointer;
     }
 
+    template <typename T>
+    WINRT_IMPL_NOINLINE void check_cast_result(T* from WINRT_IMPL_SOURCE_LOCATION_ARGS)
```

Comment:
Seems like this should be in the `impl` namespace.

Reply:
Moved to impl namespace.